ios/ReactAirplay/Modules/RAAirplayConnectivityContext.swift

```diff
@@ -28,22 +28,35 @@ class RAAirplayConnectivityContext: RAEventEmitter {
     return false
   }
 
-  @objc(fetchAirplayConnectivityWithResolver:rejecter:)
-  func fetchAirplayConnectivity(
+  @objc(fetchAirplayRoutesWithResolver:rejecter:)
+  func fetchAirplayRoutes(
     resolver resolve: RCTPromiseResolveBlock,
     rejecter reject: RCTPromiseRejectBlock
   ) {
-    resolve(getAirplayConnectivity())
+    resolve(getAirplayRoutes())
   }
 
-  private func getAirplayConnectivity() -> Bool {
+    private func getAirplayRoutes() -> [[String: Any]] {
     let session = AVAudioSession.sharedInstance()
-    let airplayConnected = session.currentRoute.outputs.contains { $0.portType == .airPlay }
-
-    return airplayConnected
+    let routes = session.currentRoute.outputs.map { route in [
+      "portName": route.portName,
+      "portType": route.portType,
+      "channels": route.channels?.map { channel in [
+        "channelName": channel.channelName,
+        "channelNumber": channel.channelNumber,
+        "owningPortUID": channel.owningPortUID,
+        "channelLabel": channel.channelLabel,
+      ]
+      },
+      "uid": route.uid,
+      "hasHardwareVoiceCallProcessing": route.hasHardwareVoiceCallProcessing,
+    ]
+    }
+
+    return routes
   }
 
   @objc private func handleRouteChange() {
-    sendEvent(withName: RAEvent.airplayConnectivityChanged.rawValue, body: getAirplayConnectivity())
+    sendEvent(withName: RAEvent.airplayConnectivityChanged.rawValue, body: getAirplayRoutes())
```

Comment:
The event should also be named `avAudioRouteSessionRoutesChanged`

Reply:
I've renamed the event and also the associated identifiers on the Swift and React Native side.